src/net.h

```diff
@@ -1197,8 +1197,19 @@ class CConnman
 
     std::unique_ptr<CSemaphore> semOutbound;
     std::unique_ptr<CSemaphore> semAddnode;
+
+    /**
+     * Maximum number of automatic connections permitted, excluding manual
+     * connections but including inbounds. May be changed by the user and is
+     * potentially limited by the number of available file descriptors.
```

Comment:
96904e8df098ba35cc5b9b8a089e472f729c0b6a

nit: Perhaps I'd be more generic saying something-something operating system limits

Reply:
I've mentioned both reasons now (the generic and the specific one).